extensions/doctrine/Numeric/Rand.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Ik snap niet helemaal wat je daarmee bedoeld? Je hebt in een module gewoon een namespace, waar je classes in kwijt kunt. Hierna kun je gewoon via de doctrine config de extensie inladen.